index.html

```diff
@@ -1,78 +1,160 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <link rel="stylesheet" href="style.css">
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Capstone</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"> 
+    <title>Capstone</title>
+    <link rel="stylesheet" href="style.css">
 </head>
 <body>
   <nav class="nav-box">
-    <div class="hamburger">
+    <div class="grey-bar">
+      <ul>
+        <li><img src="Images/facebook.png" alt="facebook"></li>
+        <li><img src="Images/twitter.png" alt="twitter"></li>
+        <li>English</li>
+        <li>My page</li>
+        <li>Logout</li>
+      </ul>
+    </div>
+    <div class="hamburger" onclick="toggleMenu()">
       <span></span>
       <span></span>
       <span></span>
     </div>
-    
-
+    <div class="dropdown-menu">
+      <a href="#home">Home</a>
+      <a href="#services">Services</a>
+      <a href="#musicians">Resident-Musicians</a>
+       <a href="about.html">About</a>
+    </div>
+    <div class="desktop-bar">
+      <div class="logo">
+        <a href="index.html"><img src="Images/Logo.png" alt="logo"></a>
+      </div>
+    <div class="dropdown-desktp">
+      <a href="#home">Home</a>
+      <a href="#services">Services</a>
+      <a href="#musicians">Resident-Musicians</a>
+       <a href="about.html">About</a>
+    </div>
+    </div>
   </nav>
-  <section>
+  <section id="home">
     <div class="whole-body1">
-    
-      <div class="div-1">
-          <h1>Welcome to Groovy studios.</h1>
+      <div class="lotainer">
+        
       </div>
       <div class="div-2">
+          <h1>Welcome to Groovy studios.</h1>
+      </div>
+      <div class="div-1 ">
         <h3>"Experience Professional-grade Sound: Record Your Next Hit with us."</h3>
       </div>
       <div class="div-3">
         <p>"Create Your Next Masterpiece in Our Professional Recording Studio. Our Experienced Engineers and Top-of-the-Line Equipment Will Bring Your Musical Vision to Life. From Recording to Mixing and Mastering, We Offer a Full Suite of Services to Elevate Your Sound. Join Our Community of Musicians and Unleash Your Creativity Today."</p>
       </div>
       <div class="visit-time">
```

Comment:
- [ ] To make your site more beautiful and create a good user experience please reduce the `font-size` of the paragraph to match that of the provided design.
<table>
  <tr>
    <td align="middle">Your design</td>
     <td align="middle">Provided design</td>
  </tr>
  <tr>
    <td align="top">
<img width="1425" alt="Screenshot 2023-04-24 at 22 36 49" src="https://user-images.githubusercontent.com/107192076/234131984-c23de185-2411-40e5-8367-a99e935e198d.png">

</td>
    <td align="top">
<img width="1080" alt="Screenshot 2023-04-24 at 22 38 02" src="https://user-images.githubusercontent.com/107192076/234132012-19c891e3-c7a0-4483-b201-fe8ce6e18d06.png">

</td>
  </tr>
 </table>